docs/_resources/changelog.md

```diff
@@ -98,12 +106,13 @@ Legend:
 >    - Reports will still be released as PBIX files so this change should not impact end users.
 >    - Visualizations are not being switched to [Power BI Enhanced Report (PBIR)](https://learn.microsoft.com/power-bi/developer/projects/projects-report#pbir-format) format yet due to functional limitations that would impact end users (as of June 2024).
 > 4. Optimized [Data ingestion report](../_reporting/power-bi/data-ingestion.md) queries to reduce memory footprint and load faster.
-> 5. Updated the [DQ page in the Cost summary report](../_reporting/power-bi/cost-summary.md#dq) to identify empty ChargeDescription rows.
-> 6. Updated the [DQ page in the Cost summary report](../_reporting/power-bi/cost-summary.md#dq) to identify potentially missing rounding adjustments.
+> 5. Updated the [Cost summary DQ page](../_reporting/power-bi/cost-summary.md#dq) to identify empty ChargeDescription rows.
+> 6. Updated the [Cost summary DQ page](../_reporting/power-bi/cost-summary.md#dq) to identify potentially missing rounding adjustments.
 >    <blockquote class="warning" markdown="1">
 >       _We are investigating an issue where we are missing rounding adjustments since May 2024. We do not yet know the cause of this issue._
 >    </blockquote>
 > 7. Simplified formatting for the `BillingPeriod` and `ChargePeriod` measures in Power BI.
+> 8. Changed the [Cost summary Purchases page](../_reporting/power-bi/cost-summary.md#purchases) and [Rate optimization Purchases page](../_reporting/power-bi/rate-optimization.md#purchases) to use PricingQuantity instead of Usage/ConsumedQuantity and added the PricingUnit column.
```

Comment:
```suggestion
> 8. Changed the [Cost summary Purchases page](../_reporting/power-bi/cost-summary.md#purchases) and [Rate optimization Purchases page](../_reporting/power-bi/rate-optimization.md#purchases) to use PricingQuantity instead of Usage/ConsumedQuantity and added the PricingUnit column.
> 9. Improved error handling for derived savings columns in the CostDetails query.
```